third-party/traefik/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@kubernetes-models/traefik",
+  "version": "2.3.0",
```

Comment:
Changeset will bump the version automatically.

```suggestion
  "version": "0.0.0",
```

Reply:
Sorry for the delay, I've made the changes you requested.